src/ess/reduce/streaming.py

```diff
@@ -174,6 +174,66 @@ def clear(self) -> None:
         self._values = []
 
 
+class MinAccumulator(Accumulator):
+    """Keeps the minimum value seen so far.
+
+    Only support scalar values.
```

Comment:
```suggestion
    Only supports scalar values.
```
Same for `MaxAccumulator`.

But this is not actually true. `_do_push` calculates the min / max of the value. So this supports any shape of data.

Reply:
I thought it would raise for non-scalars?

Reply:
It will raise, because it's just using python built-in `min` and `max`.
Should we filter it in advance...?

Reply:
No? The code uses operations along the lines of
```python
min(sc.data.table_xyz(10).data.min(), sc.scalar(10, unit='K'))
```
which works fine.

Reply:
> But this is not actually true. _do_push calculates the min / max of the value. So this supports any shape of data.

No, it raises if you pass 2 non-scalar variables in the `min` and `max`.

Reply:
But that is not what your code does. It always passes two results of `.min()` or `.max()` to the builtin `min` and `max` functions. So the inputs will always be scalar.

Reply:
It shouldn't do `.min` and `.max` on the input value since it'll be surprising if the values have different shape with/without the accumulator.

Reply:
Not a request to change, but note that one could use `scipp.reduce([a,b]).min()` to preserve input value shapes.